generators/docker-compose/index.js

```diff
@@ -0,0 +1,264 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+var shelljs = require('shelljs');
+var crypto = require('crypto');
+var _ = require('lodash');
+
+module.exports = yeoman.generators.Base.extend({
+    initializing: {
+        printJHipsterLogo: function() {
+            this.log(' \n' +
+                chalk.green('        ██') + chalk.red('  ██    ██  ████████  ███████    ██████  ████████  ████████  ███████\n') +
+                chalk.green('        ██') + chalk.red('  ██    ██     ██     ██    ██  ██          ██     ██        ██    ██\n') +
+                chalk.green('        ██') + chalk.red('  ████████     ██     ███████    █████      ██     ██████    ███████\n') +
+                chalk.green('  ██    ██') + chalk.red('  ██    ██     ██     ██             ██     ██     ██        ██   ██\n') +
+                chalk.green('   ██████ ') + chalk.red('  ██    ██  ████████  ██        ██████      ██     ████████  ██    ██\n'));
+            this.log(chalk.white.bold('                            http://jhipster.github.io\n'));
+            this.log(chalk.white('Welcome to the JHipster Docker Compose Sub-Generator '));
+            this.log(chalk.white('Files will be generated in folder: ' + chalk.yellow(this.destinationRoot())));
+        },
+
+        checkDocker: function() {
+            var done = this.async();
+
+            shelljs.exec('docker -v', {silent:true},function(code, stdout, stderr) {
+                if (stderr) {
+                    this.log(chalk.yellow.bold('WARNING!') + ' docker is not found on your computer.\n' +
+                        '         Read http://docs.docker.com/engine/installation/#installation\n');
+                }
+                done();
+            }.bind(this));
+        },
+
+        checkDockerCompose: function() {
+            var done = this.async();
+
+            shelljs.exec('docker-compose -v', {silent:true}, function(code, stdout, stderr) {
+                if (stderr) {
+                    this.log(chalk.yellow.bold('WARNING!') + ' docker-compose is not found on your computer.\n' +
+                        '         Read https://docs.docker.com/compose/install/\n');
+                }
+                done();
+            }.bind(this));
+        },
+
+        loadConfig: function() {
+
+            this.defaultAppsFolders = this.config.get('appsFolders');
```

Comment:
this would make the yo-rc file of existing app look bad, may be we should save the config under a new file using http://yeoman.io/generator/Storage.html `new Storage(name, fs, configPath)` method